modules/gapi/cmake/DownloadOneVPL.cmake.in

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(oneVPL-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(oneVPL
```

Comment:
We don't use `ExternalProject_Add` in OpenCV.

`find_package(VPL)` should be enough.

Reply:
I thought  about automatic "download & build" feature for oneVPL in case it not installed or not found  in the system.
It completely reduces required configuration steps for developers: no need worries about how to get oneVPL, configure, sourcing PATH and so on

If `ExternalProjecy_Add` is abandoned cmake feature  in  opencv -  Then I'll remove it and we loose "download & build" for oneVPL. 
@alalek  Please respond - did I understand you comment correctly?

Reply:
Consider oneVPL dependency as prerequisite.

Reply:
done. entire file removed from PR